src/main/java/org/apache/commons/crypto/random/OsCryptoRandom.java

```diff
@@ -74,7 +76,7 @@ public OsCryptoRandom(final Properties props) {
 
         try {
             close();
-            this.stream = new FileInputStream(randomDevFile);
+            this.stream = Files.newInputStream(Paths.get(randomDevFile.getPath()));
```

Comment:
I'm not 100% sure about this change.

Reply:
This is coded in a pretty obtuse manner, if the point is to use NIO, then start with a Path instead of a File.

In general, it feels to me like you need to step back and look at the big picture instead of applying whatever micro changes your IDE offers (or PMD, and so on).

Reply:
OK @garydgregory 
I close the PR and I will review the code
TY.